src/collective/behavior/amp/validators.py

```diff
@@ -40,3 +40,15 @@ def is_valid_logo(value):
             u'Image should be no wider than 600px, and no taller than 60px.'))
 
     return True
+
+
+def is_valid_tags(value):
+    url = 'www.cartacapital.com.br/internacional/jean-luc-melenchon-o-esquerdista-que-sacode-a-campanha-presidencial-francesa/@@amp'
```

Comment:
then what is your suggestion? I don't have any website with AMP view. Where should I test?
That's why I hardcoded it just for the sake checking.... later we can mould to our needs